layersvt/device_simulation.md

```diff
@@ -1,52 +1,111 @@
-# Device Simulation Layer (DevSim)
-The Device Simulation Layer is a Vulkan layer that can modify the results of Vulkan PhysicalDevice queries based on a configuration file,
-thus simulating a different device than is actually in a system.
-A goal is to simplify application testing for a wide range of simulated devices, without actually requiring a physical copy of every device.
+# VK_LAYER_LUNARG_device_simulation (DevSim)
+The goal of this layer is to simplify application testing on a wide range of simulated device capabilities, without requiring an actual physical copy of every device.
 
-The DevSim layer loads a configuration file at application startup, which specifies the capabilities of the simulated device.
+The Device Simulation Layer is a Vulkan layer that can modify the results of Vulkan PhysicalDevice queries based on a JSON configuration file, thus simulating a different device than is actually in a system.
```

Comment:
I've always had difficulty with "simulating a device".   Something like "simulating some of a device's capabilities" is much better.  In this case, this sentence might be better ending as:

... thus simulating some of the capabilities of a device by overriding the capabilities of the actual device under test.

If you agree that "simulating a device's capabilities" is better than, "simulating a device", then it may be good to look in other docs and in the source code comments for other instances of "simulating a device" that could be replaced with the more precise wording.